mmv1/templates/terraform/constants/cloud_run_service.go.erb

```diff
@@ -65,6 +65,7 @@ func cloudrunLabelDiffSuppress(k, old, new string, d *schema.ResourceData) bool
 
 var cloudRunGoogleProvidedTemplateLabels = []string{
 	"run.googleapis.com/startupProbeType",
+	"run.googleapis.com/startup-cpu-boost",
```

Comment:
This is used in template annotations instead of template labels. I think you need to update `cloudrunTemplateAnnotationDiffSuppress`

Reply:
Fixed and refactored to match the label matching.